.github/actions/spelling/excludes.txt

```diff
@@ -37,27 +46,95 @@ ignore$
 \.lock$
 \.map$
 \.min\..
+\.mo$
 \.mod$
 \.mp[34]$
 \.o$
 \.ocf$
 \.otf$
+\.p12$
+\.parquet$
 \.pdf$
 \.pem$
+\.pfx$
 \.png$
 \.psd$
 \.pyc$
 \.pylintrc$
+\.qm$
 \.s$
+\.sig$
+\.so$
 \.svgz?$
+\.sys$
 \.tar$
+\.tgz$
 \.tiff?$
 \.ttf$
 \.wav$
 \.webm$
 \.webp$
 \.woff2?$
+\.xcf$
 \.xlsx?$
+\.xpm$
+\.xz$
 \.zip$
+^localization/japanese/
```

Comment:
This was a manual choice on my part. check-spelling atm ignores Japanese content and thus doesn't realize that the files are more Japanese than English -- if it did, it'd suggest skipping them.

It's possible to run prerelease with support for non-English languages, and I might try it to see how this behaves, but for the time being...